CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+- BUGFIX: Revert changes made on v5.1.1. 
+    - The issues addressed by those changes, were non-issues and result of misuse of the APIs.
```

Comment:
Is there anything we need to clarify in the glean book?

Reply:
I will check that later today. Might be worth noting that the metrics are module private, although that miht be obvioust o a more experienced Swift programmer.

Reply:
The `internal` access is indeed "module internal/private", and might be worth calling out. Had I been thinking about that and how Firefox iOS was structured, we wouldn't have gone this route. I would also be glad to see more documentation/comments to help guide us the next time something like this comes up. Feel free to file a bug around adding some clarification to using Glean in a project that has multiple targets/modules in Xcode, definitely a little more challenging getting everything just right than I expected.